src/components/SøknadsSteg/Steg/Steg.tsx

```diff
@@ -149,6 +149,34 @@ const Steg: React.FC<ISteg> = ({ tittel, children, erSpørsmålBesvart }) => {
                     </Hovedknapp>
                 </div>
             )}
+            <Modal
+                className={'steg__modal'}
+                isOpen={åpenModal}
+                onRequestClose={() => veksleModal()}
+                closeButton={true}
+                contentLabel="avbryt-søknad-modal"
+                shouldCloseOnOverlayClick={true}
+            >
+                <div className={'modal-container'}>
+                    <Informasjonsbolk
+                        tittel={
+                            <Undertittel>Er du sikker at vil avbryte søknadprosessen?</Undertittel>
+                        }
+                    >
+                        <Normaltekst>OBS: Hvis du avbryter vil søknaden bli slettet.</Normaltekst>
```

Comment:
Kanskje "Hvis du avbryter vil innholdet i søknaden bli slettet" er bedre? Vet ikke, det hørtes bare så brutalt ut 🤔

Reply:
Enig, kan virke litt for skremmende 🙆‍♂️